hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -519,18 +481,9 @@ private void mapModelChangesToMutable(Token token, MerkleToken mutableToken) {
 		mutableToken.setKycKey(token.getKycKey());
 		mutableToken.setFeeScheduleKey(token.getFeeScheduleKey());
 		mutableToken.setPauseKey(token.getPauseKey());
-
-		mutableToken.setMaxSupply(token.getMaxSupply());
```

Comment:
How can we remove these? Won't this break `TokenUpdate` and `TokenFeeScheduleUpdate`?